tasks/coverage/snapshots/parser_babel.snap

```diff
@@ -628,6 +628,74 @@ Expect to Parse: tasks/coverage/babel/packages/babel-parser/test/fixtures/typesc
  10 │ 
     ╰────
 
+Expect to Parse: tasks/coverage/babel/packages/babel-parser/test/fixtures/typescript/declare/eval-dts/input.ts
+
+  × Cannot assign to 'eval' in strict mode
+   ╭─[babel/packages/babel-parser/test/fixtures/typescript/declare/eval-dts/input.ts:4:25]
+ 3 │     export namespace ns {
+ 4 │         export function eval(): void;
+   ·                         ────
+ 5 │         export function arguments(): void;
+   ╰────
```

Comment:
this is a bug that we should fix in a follow up. Since it only affects `.d.ts`, i think it's ok to merge as-is